packages/framework/undo-redo/src/index.ts

```diff
@@ -4,89 +4,77 @@
- * the valueChanged event of the SharedMap. This handler will never close the
- * current operation on the stack. This is a fairly simple handler, and a good
- * example to look at for understanding how IRevertible objects should work.
- *
- * <h3>Shared Segment Sequence Undo Redo Handler</h3>
- *
- * The SharedSegmentSequenceUndoRedoHandler generates IRevertible objects,
- * SharedSegmentSequenceRevertible for any SharedSegmentSequence based
- * distributed data structures like SharedString.
- *
- * This handler pushes an SharedSegmentSequenceRevertible for every local Insert,
- * Remove, and Annotate operations made to the sequence. The objects are created
- * via the sequenceDelta event of the sequence. Like the SharedMapUndoRedoHandler
- * this handler will never close the current operation on the stack.
- *
- * This handler is more complex than the SharedMapUndoRedoHandler. The handler
- * itself batches the SharedSegmentSequence changes into the smallest number of
- * IRevertible objects it can to minimize the memory and performance overhead on
- * the SharedSegmentSequence of tracking changes for revert.
- *
- * <h3>Shared Segment Sequence Revertible</h3>
- *
- * The SharedSegmentSequenceRevertible does the heavy lifting of tracking and
- * reverting changes on the underlying SharedSegmentSequence. This is
- * accomplished via TrackingGroup objects. A TrackingGroup creates a bi-direction
- * link between itself and the segment. This link is maintained across segment
- * movement, splits, merges, and removal. When a sequence delta event is fired
- * the segments contained in that event are added to a TrackingGroup. The
- * TrackingGroup is then tracked along with additional metadata, like the delta
- * type and the annotate property changes. From the TrackingGroup's segments we
- * can find the ranges in the current document that were affected by the original
- * change even in the presence of other changes. The segments also contain the
- * content which can be used. With the ranges, content, and metadata we can
- * revert the original change on the sequence.
- *
- * As called out above, there is some memory and performance overhead associated
- * with undo redo. This overhead is from the TrackingGroup. This overhead
- * manifests in a few ways:
- *
- * - Removed segments in a TrackingGroup will not be garbage collected from the
- * backing tree structure.
- *
- * - Segments can only be merged if they have all the same TrackingGroups.
- *
- * This object minimizes the number of TrackingGroups created, so this overhead
- * is very low. This undo redo infrastructure is entirely in-memory so it does
- * not affect other users or sessions. If custom IRevertible objects use
- * TrackingGroups this overhead should be kept in mind to avoid possible
- * performance issues.
+ * This library provides an implementation of an in-memory undo redo stack, as well as handlers for the
```

Comment:
Note: the existing line wrapping was very narrow, and my changes affected the layout enough that I went ahead and just re-formatted it all. Sorry for the annoying diff :/

The only intentional changes below are:
1. Moving most of the detailed information into a `@remarks` block
2. Removing the unsupported use of HTML syntax 
3. Using `{@link}`s where appropriate
4. Using `code` syntax for code references elsewhere.

Reply:
And I think I fixed a typo or two.